domain/src/main/java/com/teamagam/gimelgimel/domain/layers/ProcessNewVectorLayerInteractor.java

```diff
@@ -59,15 +59,13 @@
     protected Iterable<SubscriptionRequest> buildSubscriptionRequests(
             DataSubscriptionRequest.SubscriptionRequestFactory factory) {
         DataSubscriptionRequest dataSubscriptionRequest = factory.create(
-                buildObservable());
+                Observable.just(null),
```

Comment:
Problematic because of current class operations structure.
It'll be better to leave it this way.